scripts/build_package.py

```diff
@@ -90,6 +90,24 @@ def build_ld_logger(ld_logger_path, env, arch=None, clean=True, silent=True):
         return ret
 
 
+def build_plist_to_html(plist_to_html_path, env, clean=True, silent=True):
```

Comment:
Please move this out to the `Makefile` as a new target and the package target should depend on this.